src/pages/Dashboard/DashboardSidebar.tsx

```diff
@@ -1,17 +1,35 @@
 import { Flex } from '@chakra-ui/react'
 import { useSelector } from 'react-redux'
+import { IOSPromoCards } from 'components/Promo/iOSPromoCards'
 import { PromoCard } from 'components/Promo/PromoCard'
+import type { PromoItem } from 'components/Promo/types'
 import { EligibleCarousel } from 'pages/Defi/components/EligibleCarousel'
 import { selectFeatureFlags } from 'state/slices/selectors'
 
 import { RecentTransactions } from './RecentTransactions'
 import { TradeCard } from './TradeCard'
 
+const promoData: PromoItem[] = [
+  {
+    title: 'Earn 3.15% APY on FOX',
+    body: 'Your FOX is put to work across different DeFi protocols to earn the best yield possible.',
```

Comment:
do we want to translate this?

Reply:
ehhh its an old ad that i'm not sure we are going to run again

Reply:
Might as well just remove it at this point....